ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,111 @@
+
+
+def resize_icon(source_path, output_dir, filename, size):
+    """Создает иконку нужного размера из исходного изображения"""
+    try:
+        # Загружаем исходное изображение
+        with Image.open(source_path) as img:
+            # Конвертируем в RGBA если нужно
+            if img.mode != "RGBA":
+                img = img.convert("RGBA")
+
+            # Изменяем размер с сохранением качества
```

Ruff (0.13.1)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)